FirebaseSessions/Sources/ApplicationInfo.swift

```diff
@@ -16,7 +16,16 @@
 import Foundation
 
 @_implementationOnly import FirebaseCore
-@_implementationOnly import GoogleUtilities
```

Comment:
We've been adding `@_implementationOnly` to our imports of other libraries. Necessary or no? I don't see others doing it when I search for `import GoogleUtilities_` in swift code

Reply:
Gonna fix that in this PR for Sessions SDK

Reply:
Interesting - it gets unhappy when I do that for the ObjC target
![Screen Shot 2022-12-14 at 1 17 13 PM](https://user-images.githubusercontent.com/555046/207674809-6b72f1a7-05e3-446a-bbeb-4c0007ed83a1.png)

Reply:
I think it is because using `@_implementationOnly XYZ` implies you do not intend to import the API from module XYZ and expose it publicly. Does `description` need to be public? If `firebase_appaquality_sessions_eventType` needs to be used outside of the `Sessions` target, than don't worry about using `@_implementationOnly` for this particular import.

Reply:
`description` needs to be public to the Sessions SDK, but not outside it. In fact, nothing in the Sessions SDK (yet) needs to be public because it's just going to be a dependency of Crash / Perf. So `firebase_appaquality_sessions_eventType` should only be public to the Sessions SDK, but not users of the Sessions SDK.

I'm going to submit this PR for now and maybe we can follow up later on tightening up access controls to the SDK.